The Times and Sunday Times.js

```diff
@@ -0,0 +1,174 @@
+{
+	"translatorID": "53f8d182-4edc-4eab-b5a1-141698a10101",
+	"label": "The Times and Sunday Times",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://www\\.thetimes\\.co\\.uk/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2017-07-05 06:39:28"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 Philipp Zumstein
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	if (url.indexOf('/edition/')>-1 || url.indexOf('/article/')>-1) {
+		return "newspaperArticle";
+	}
+}
+
+
+function doWeb(doc, url) {
+	scrape(doc, url);
+}
+
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	//translator.setDocument(doc);
+	
+	translator.setHandler('itemDone', function (obj, item) {
+		var date = ZU.xpathText(doc, '//time[contains(@class, "Dateline")]');
+		if (date) {
+			item.date = ZU.strToISO(date);
+		}
+		item.ISSN = "0140-0460";
+		item.publicationTitle = ZU.xpathText(doc, '//p[contains(@class, "Meta-content")]/span[contains(@class, "Publication")]');
+		item.url = ZU.xpathText(doc, '//link[@rel="canonical"]/@href') || url;
+		item.section = ZU.xpathText(doc, '//article[@id="article-main"]/@data-article-sectionname');
+		item.complete();
+	});
+
+	translator.getTranslatorObject(function(trans) {
+		trans.itemType = "newspaperArticle";
+		trans.doWeb(doc, url);
+	});
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://www.thetimes.co.uk/edition/scotland/rare-animals-among-body-count-at-scottish-zoos-htpkmkfvx",
+		"items": [
+			{
+				"itemType": "newspaperArticle",
+				"title": "Rare animals among body count at Scottish zoos",
+				"creators": [
+					{
+						"firstName": "Mark",
+						"lastName": "Macaskill",
+						"creatorType": "author"
+					}
+				],
+				"date": "2017-07-02",
+				"ISSN": "0140-0460",
```

Comment:
I think the Sunday times should be `0956-1382` see https://en.wikipedia.org/wiki/The_Sunday_Times